pandas/tests/groupby/test_categorical.py

```diff
@@ -1669,3 +1669,27 @@ def test_categorical_transform():
     expected["status"] = expected["status"].astype(delivery_status_type)
 
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("func", ["first", "last"])
```

Comment:
can you parameterize this over observed=True/False (I am not sure if its the same result or not),

Reply:
Done